test/main.js

```diff
@@ -24,6 +26,8 @@ var SelfInsertingView = View.extend({
     render: function () {}
 });
 
+viewCompliance.view(test, ViewSwitcher, { view: new makeTestView()() });
```

Comment:
I wonder if this has to do with `ViewSwitcher` not being a instance of `AmpersandView`/`AmpersandState`.

Should that even matter? Can regular, non-ampersand objects achieve ampersand-view-compliance? I'll have to think about this some more (when I have some spare time on the weekend).

In the meantime, would it be worth having `ViewSwitcher` extend `AmpersandView`?